frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -29,6 +29,16 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
    */
   challenge: any;
 
+  /**
+   * Leaderboard object
+   */
+  leaderboard: any;
+
+  /**
+   * 
```

Comment:
@vinceli1004 can you add a docstring here as well

Reply:
Adressed